exporter/splunkhecexporter/buffer.go

```diff
@@ -113,10 +125,18 @@ func (c *cancellableGzipWriter) Reset() {
 	c.rawLen = 0
 }
 
-func (c *cancellableGzipWriter) Close() error {
+func (c *cancellableGzipWriter) Flush() error {
 	return c.innerWriter.Close()
```

Comment:
`gzip.Writer` actually also has a `Flush` method, would that be more adequate here?